### PR TITLE
Fix service to work with private channels and minor refactorings

### PR DESCRIPTION
**Pull request by ruippeixotog**
More information about the API endpoints involved:
- https://api.slack.com/methods/channels.list
- https://api.slack.com/methods/groups.list

### CODE REVIEW

**Comment by ruippeixotog** [comment]
@DDuarte, check this code too when you have the time ;)

File: door-open-service/node-backend/app.js [hunk 1]
```diff
@@ -13,7 +13,7 @@ const _ = require('underscore');
 const BellServer = require('./bellServer');
 const backend = require('./backend');
 
-const CHANNEL = '<CHANNEL_NAME>';
+const CHANNEL = { name: '<CHANNEL_NAME>', private: true };
```
Comment (queimadus) [inline door-open-service/node-backend/app.js:16]: Would you consider extracting these variables to an external json file?

Comment (ruippeixotog) [inline door-open-service/node-backend/app.js:16]: Sure! A big refactoring with more than just extracting the variables is scheduled :) But I'll open a dedicated PR just for that; right now the immediate goal is to implement the necessary to add quick support for more doors.

File: door-open-service/node-backend/app.js [hunk 2]
```diff
@@ -33,31 +33,35 @@ const bot = new SlackBot({
     name: BOT_NAME
 });
 
-bot.on('message', rawData => {
-  console.log('bot: ' + JSON.stringify(rawData));
+const channelInfoPromise = CHANNEL.private ?
+  bot.getGroup(CHANNEL.name) :
```
Comment (jcazevedo) [inline door-open-service/node-backend/app.js:37]: A private channel is a group?

**Comment by jcazevedo** [comment]
LGTM

**Pull request merged**